README.md

```diff
@@ -44,25 +44,116 @@ This repository is maintained using [OET's soft-fork strategy](https://open-ener
 
-    mamba env create -f environment.yaml
+```sh
+pixi shell
+```
 
-Activate the newly created `{{project_short_name}}` environment:
+>[!NOTE]
+>If `pixi` isn't working, you can install from one of the fallback `conda` environment files found in `envs`.
+>For more details see [the PyPSA-Eur installation guide](https://pypsa-eur.readthedocs.io/en/latest/installation.html).
+
+### Sharing pixi environment across copies of the same cloned project
+
+`pixi` works on the concept of per-project dependency management.
+Unlike `conda`, this means that you cannot generally use the same environment when you switch between local directories.
+If you have several copies of this repository cloned locally (e.g., to develop features in parallel), then you can avoid duplicating the working environment (and the space on disk it entails) by calling:
+
+```sh
+pixi shell --manifest-path /path/to/main/project/clone/pixi.toml -e {{ project_name }}
+```
+
+Where one local clone is designating as the "main" clone, from which all other clones will inherit their `pixi` environment.
+
+>[!WARNING]
+>Although it reduces overall space on disk, sharing a `pixi` environment is not actually recommended.
+>This is due to possible discrepancies in the dependencies that are defined between the different clones.
+
+### Extra soft-fork dependencies
+
+If you add dependencies to your project, we recommend you add them to a [new `pixi` environment](https://pixi.sh/v0.21.1/features/multi_environment/#feature-environment-set-definitions).
+For instance, if you need access to `plotly`, want to pin the version of gurobi you are using, and want to add a PyPI dependency:
+
+```sh
+pixi add -f {{ project_short_name }} "gurobi<13" "plotly"
+pixi add -f {{ project_short_name }} --pypi pypsa-explorer
+```
+
+This will create these entries in your `pixi.toml`
+
+```toml
+[feature.{{ project_short_name }}.pypi-dependencies]
+pypsa-explorer = "*"
+
+[feature.{{ project_short_name }}.dependencies]
+gurobi = "<13"
+plotly = "*"
+```
+
+Then, you can create an environment from this feature in `pixi.toml`:
+
+```toml
+[environments]
+...
+{{ project_short_name }} = [{{ project_short_name }}]
+```
+
+These dependencies will be combined with the core PyPSA-Eur dependencies and can be accessed by calling:
+
+```sh
+pixi shell -e {{ project_short_name }}
+```
+
+#### Updating CI tests
+
+To run CI tests using your environment you should add the `test` feature to it and create test tasks for your environment, e.g.:
+
+```toml
+[feature.{{ project_short_name }}.tasks]
+{{ project_short_name }}-test = """
+	snakemake --configfile config/config.{{ project_short_name }}.default.yaml -n &&
+    """
+[environments]
+...
+{{ project_short_name }} = ["test", {{ project_short_name }}]
+```
+
+And then update `.github/workflows/test.yaml` to run that test:
+
+```yaml
+- name: Run project-specific snakemake test workflows
+  run: |
+    pixi run {{ project_short_name }}-test
```

Comment:
Why not:
```
pixi run -e tyndp test
```
like with the unit tests?

Reply:
Hmm .. no, i mean if you added a task with the name of `test` to the `tyndp` feature, would you invoke both tasks when you choose the tyndp environment (the feature one and the global one)? but maybe i am missing the point.